### ISSUE TITLE
Without proper notice or anything features were removed

### ISSUE THREAD

**Issue by hkmsadek** [unlayer/react-email-editor#427]
This is so ridiculous and annoying to remove features from free plan and make it expensive now. even simple copy paste doesn't work

**Comment by omerzahidbajwa** [comment]
Please let me know if you pass in a "projectId" during initialization. Unlayer has just rolled out new billing plans, and many features are paid. If you are an old user and have an old projectId available, you can still utilize the old "Free (Legacy)" plan feature sets.

Pass in the projectId within your editor, and make sure to add your live domain to the "Allowed Domains" list which you can find under project->settings->deployment.

Let me know if this works

Note:
Simple copy-paste still works (without any formatting) however "clean paste: basic (formatted paste)" is a paid feature within the newly launched Unlayer pricing plans (https://unlayer.com/pricing).

**Comment by nickrobinson352** [comment]
I tried passing in the old projectid and paste is still not working.

**Comment by p3rv3z** [comment]
Paste feature is working for me. 🩷 u guys!

**Comment by nickrobinson352** [comment]
How did you get it to work ?

**Comment by PaulSong213** [comment]
Even the mobile design mode is not free now

**Comment by ericsukalac** [comment]
> Even the mobile design mode is not free now

Agreed. That's frustrating that the documentation still doesn't reflect this - it simply stopped being available.

https://docs.unlayer.com/docs/device-management

**Comment by Keshavdulal** [comment]
Found it a hard-way after debugging the issue since yesterday.
No mention on docs.
https://unlayer.com/pricing
![Image](https://github.com/user-attachments/assets/20e5606f-f051-4e38-879a-be0fe4730ad1)

We were using custom storage, let me repeat "CUSTOM STORAGE", and it stopped working a few days ago silently and guess who informed us.
Damn well Our clients did!

And Its not that we wouldn't pay for it, as we are using it commercially, but $750/mo to get that feature! Come on!
We South Asians can build the whole library for much less and we are very sarcastic as well.

Sadly, I am looking at alternatives now.
https://github.com/zalify/easy-email-editor

**Comment by pnp-os** [comment]
@Keshavdulal Check this temporary workaround

https://github.com/unlayer/react-email-editor/issues/435#issuecomment-2642178833
